apps/vscode/vscode.py

```diff
@@ -223,6 +223,14 @@ def split_window_vertically():
     def split_window():
         actions.user.vscode("workbench.action.splitEditor")
 
+    def split_number(index: int):
+        """Navigates to a the specified split"""
+        if index < 9:
+            if is_mac:
+                actions.key(f"cmd-{index}")
+            else:
+                actions.key(f"ctrl-{index}")
```

Comment:
From the meetup: is there a VS Code action that we can use to do this?

It looks like @AndreasArvidsson  added `andreas.openEditorAtIndex` in his extension which makes me suspect it is not in the base VS Code

Reply:
Oh looks like there is `workbench.action.openEditorAtIndex1` throuh `workbench.action.openEditorAtIndex9` :p. Shall we use that instead?

Reply:
The main purpose of my implementation is that it supports tabs higher than 9 as well as negative numbers (counting from the back).

Reply:
Does it make sense for community to try to use your action if it's present?

Reply:
I don't know. Ideally it would be a different grammar that supported larger numbers and also negative numbers. For community is probably better to strive for simplicity and more widespread support.

Reply:
`workbench.action.openEditorAtIndex1` does the tab number rather than the split number, used `workbench.action.focusFirstEditorGroup` etc.